source/wp-content/themes/wporg-developer-blog/templates/category.html

```diff
@@ -1,33 +1,65 @@
-<!-- wp:template-part {"slug":"header","className":"site-header-container"} /-->
+<!-- wp:template-part {"slug":"header"} /-->
 
-<!-- wp:spacer {"height":"30px"} -->
-<div style="height:30px" aria-hidden="true" class="wp-block-spacer"></div>
-<!-- /wp:spacer -->
+<!-- wp:wporg/local-navigation-bar {"backgroundColor":"charcoal-2","style":{"elements":{"link":{"color":{"text":"var:preset|color|white"},":hover":{"color":{"text":"var:preset|color|white"}}}},"position":{"type":"sticky","top":"0px"}},"fontSize":"small"} -->
+	<!-- wp:site-title {"level":0,"fontSize":"small"} /-->
 
-<!-- wp:columns {"align":"full","className":"is-style-two-column-display"} -->
-<div class="wp-block-columns alignfull is-style-two-column-display">
-	<!-- wp:column {"style":{"spacing":{"blockGap":"16px"}},"className":"is-left-column"} -->
-	<div class="wp-block-column is-left-column">
-		<!-- wp:group -->
-		<div class="wp-block-group">
-			<!-- wp:query-title {"type":"archive", "level":2} /-->
+	<!-- wp:navigation {"openSubmenusOnClick":true,"icon":"menu","overlayBackgroundColor":"charcoal-2","overlayTextColor":"white","layout":{"type":"flex","orientation":"horizontal"},"fontSize":"small","menuSlug":""} /-->
+<!-- /wp:wporg/local-navigation-bar -->
 
-			<!-- wp:term-description /-->
+<!-- wp:group {"style":{"spacing":{"padding":{"right":"var:preset|spacing|edge-space","left":"var:preset|spacing|edge-space","top":"var:preset|spacing|30","bottom":"var:preset|spacing|20"}}},"layout":{"type":"constrained"}} -->
+<div class="wp-block-group" style="padding-top:var(--wp--preset--spacing--30);padding-right:var(--wp--preset--spacing--edge-space);padding-bottom:var(--wp--preset--spacing--20);padding-left:var(--wp--preset--spacing--edge-space)">
+	<!-- wp:columns {"align":"wide","style":{"spacing":{"blockGap":{"top":"var:preset|spacing|50","left":"var:preset|spacing|80"},"padding":{"top":"var:preset|spacing|40","bottom":"var:preset|spacing|40"}}}} -->
+	<div class="wp-block-columns alignwide" style="padding-top:var(--wp--preset--spacing--40);padding-bottom:var(--wp--preset--spacing--40)">
+		<!-- wp:column {"width":"300px"} -->
+		<div class="wp-block-column" style="flex-basis:300px">
+			<!-- wp:group {"style":{"spacing":{"blockGap":"var:preset|spacing|30"}},"layout":{"type":"constrained"}} -->
+			<div class="wp-block-group">
+				<!-- wp:query-title {"type":"archive","fontSize":"heading-2"} /-->
+
+				<!-- wp:term-description /-->
+			</div>
+			<!-- /wp:group -->
 		</div>
-		<!-- /wp:group -->
-	</div>
-	<!-- /wp:column -->
+		<!-- /wp:column -->
+
+		<!-- wp:column {"width":""} -->
```

Comment:
Is this property needed?

Reply:
Nope, good catch. I designed the templates in the Editor, so that must be applied when you remove the width from a column. Interesting... 🤔 All fixed in the latest commit.